src/App.jsx

```diff
@@ -1,34 +1,23 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import { BrowserRouter, Route, Routes } from 'react-router-dom'
 import './App.css'
-
+import Nav from './components/Nav/Nav'
+import Home from './components/home/home'
+import Error from './components/Error/Error'
 function App() {
-  const [count, setCount] = useState(0)
+  
 
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+   <BrowserRouter>
+   <Nav />
+   <Routes>
+    <Route path='/' element={<Home />} />
+    <Route path='*' element={<Error />} />
+   </Routes>
+   
+   
+   
+   
+   </BrowserRouter>
```

Comment:
Should: The indentation needs to be fixed here to improve the code readability.
Also worth considering removing some of the extr new lines for the same reason.